sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveDataFrameAnalyticsSuite.scala

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark.sql.hive
```

Comment:
Thank you, Andrew! 

Using SQL statements might generate a different logical plan. Thus, we also need to do the unit test in this package org.apache.spark.sql.hive.

Sure, I should also add unit cases in the base sql package.